<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuchuangLoginForm</class>
 <widget class="QWidget" name="tuchuangLoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>s6tu.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s2tu.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pic303.com</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUser"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditpwd"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancle">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
